src/com/dhbw/MorseProject/send/Encoder.java

```diff
@@ -1,4 +1,184 @@
 package com.dhbw.MorseProject.send;
 
+import com.dhbw.MorseProject.translate.Translator;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+
+// todo comments
```

Comment:
TODOs in main branch are not common, but it should be okay as this has nothing to do with the code itself.